lib/ws-kernel.coffee

```diff
@@ -50,6 +51,20 @@ class WSKernel extends Kernel
                 stream: 'status'
             onResults?(result)
 
+        future.onStdin = (message) =>
+            if message.header.msg_type is not 'input_request'
```

Comment:
Right. No wonder I felt that I hadn't ever seen `if ... is not` in hydrogen before.